bin/setup.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+# Setup script for Artsy's use.
+
+# It assumes that the general Artsy pre-requisites are already installed.
+# Please see: https://github.com/artsy/potential/blob/master/scripts/setup
+
+trap 'ret=$?; test $ret -ne 0 && printf "Setup script failed.\n\n" >&2; exit $ret' EXIT
+
+set -e
+
+python_version=$(cat .python-version)
+if (command -v pyenv >/dev/null)
+then
+  echo "Found Pyenv."
+  if (pyenv versions | grep -Fq "$python_version")
+  then
+    echo "Found Python $python_version."
+  else
+    echo "Using Pyenv to install Python $python_version..."
+    pyenv install -s "$python_version"
+  fi
+  pyenv rehash
+  echo "Installing Poetry..."
+  pip install poetry
+  echo "Running poetry install..."
+  poetry install
+else
+  echo "Pyenv not found, will not install Python packages, for fear of polluting system-wide Python installation."
```

Comment:
We have code for installing pyenv and activating the local virtualenv in most of our setup scripts, for example [here](https://github.com/artsy/degas/blob/main/scripts/setup.sh#L32-L67).

Reply:
nice. [copied](https://github.com/artsy/fortress/pull/3/commits/ffdeaeda52dca18c4654d229128cc7b634cf0c0d).